web/src/app/modules/sugarloaf/components/smart/namespace/namespace.component.ts

```diff
@@ -51,7 +53,7 @@ export class NamespaceComponent implements OnInit, OnDestroy {
 
     this.namespaceSubscription = this.namespaceService.availableNamespaces.subscribe(
       (namespaces: string[]) => {
-        this.namespaces = namespaces;
+        this.namespaces = namespaces.sort();
```

Comment:
I thought we already sorted the list server side?

Reply:
It looks that way looking at the ui too but i couldn't find the code on server side which sorts the namespaces. May be i missed it or there are options set instead of code doing it. I'll revert it.